src/views/start/index.jsx

```diff
@@ -33,10 +33,18 @@ export default class Start extends Component {
 
   openFolderDialog = async () => {
     const queryParameter = document.getElementById('queryParameter').value;
+    let fileExtension = document.getElementById('fileExtension').value;
```

Comment:
Please try to get away from this kind of getting the value. This is plain javascript style. In React you set the value with an onChange methode directly in the element-tag in the render-method. Example can be found here: https://github.com/sprinteins/ghost/pull/45/files

Reply:
hi, try using the textbox without the *. prefix, you can type all the extensions without any spaces. use something like this: js,jsx,json. 
I'll check out the other way of getting the value.

Reply:
ah okay.  sorry - my bad

Reply:
np, i also accientally commited a log. I'll change the value readout to your suggested style and commit.